# HG changeset patch
# User Rik <rik@octave.org>
# Date 1455042875 28800
#      Tue Feb 09 10:34:35 2016 -0800
# Branch stable
# Node ID 6b0ded11d7087822702146919216c74feadf5991
# Parent  732ec49d1ec5c5dd5332f8d2364f010514b4c5c2
Fix cut/paste error in rande documentation (bug #47097).

* rand.cc (Frande): Refer to rande and exponential distribution in docstring.

diff --git a/libinterp/corefcn/rand.cc b/libinterp/corefcn/rand.cc
--- a/libinterp/corefcn/rand.cc
+++ b/libinterp/corefcn/rand.cc
@@ -633,18 +633,19 @@ DEFUN (rande, args, ,
 @deftypefnx {Built-in Function} {} rande (\"seed\", @var{v})\n\
 @deftypefnx {Built-in Function} {} rande (\"seed\", \"reset\")\n\
 @deftypefnx {Built-in Function} {} rande (@dots{}, \"single\")\n\
 @deftypefnx {Built-in Function} {} rande (@dots{}, \"double\")\n\
 Return a matrix with exponentially distributed random elements.\n\
 \n\
 The arguments are handled the same as the arguments for @code{rand}.\n\
 \n\
-By default, @code{randn} uses the @nospell{Marsaglia and Tsang}\n\
-``Ziggurat technique'' to transform from a uniform to a normal distribution.\n\
+By default, @code{rande} uses the @nospell{Marsaglia and Tsang}\n\
+``Ziggurat technique'' to transform from a uniform to an exponential\n\
+distribution.\n\
 \n\
 The class of the value returned can be controlled by a trailing\n\
 @qcode{\"double\"} or @qcode{\"single\"} argument.  These are the only valid\n\
 classes.\n\
 \n\
 Reference: @nospell{G. Marsaglia and W.W. Tsang},\n\
 @cite{Ziggurat Method for Generating Random Variables},\n\
 J. Statistical Software, vol 5, 2000,\n\
